src/core/PDF.cpp

```diff
@@ -144,10 +144,9 @@ QImage PDF::renderToImage() const
     return pdfPage_->renderToImage();
 }
 
-void PDF::getDimensions(int &width, int &height) const
+QSize PDF::getSize() const
```

Comment:
Does not work with the current implementation as it would return a reference on a temp object